examples/example-base/example-base.cpp

```diff
@@ -26,6 +26,7 @@ Slang::Result WindowedAppBase::initializeBase(const char* title, int width, int
 
     // Initialize the rendering layer.
     IDevice::Desc deviceDesc = {};
+    deviceDesc.deviceType = gfx::DeviceType::DirectX12;
```

Comment:
We can remove this line since dx12 is already the default. We need to make our examples able to take command line arguments to determine what API to use for our debugging purposes.

Reply:
Sorry, that is debugging code that slipped in.